strawberryfields/apps/graph/resize.py

```diff
@@ -256,6 +256,22 @@ def clique_grow(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
     supported. Degree-based node selection involves picking the node with the greatest degree,
     with ties settled by uniform random choice.
 
+    Example usage:
+
+    .. code-block::
+
+        >>> from strawberryfields.apps.graph import resize
+        >>> import networkx as nx
+        >>> graph = nx.complete_graph(10)
+        >>> clique = [0, 1, 2, 3, 4]
+        >>> resize.clique_grow(clique, graph)
+        [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+        >>> graph = nx.lollipop_graph(5, 1)
+        >>> graph.remove_edge(3, 4)
+        >>> clique = [0, 1, 2]
+        >>> resize.clique_grow(clique, graph, node_select="degree")
+        [0, 1, 2, 4]
```

Comment:
Note that the use of `>>>` negates the need to use `.. code-block::` and indent.